src/main.js

```diff
@@ -4,10 +4,56 @@ import {createTripFiltersTemplate} from "./view/trip-filters";
 import {createTripSortTemplate} from "./view/trip-sort";
 import {createTripEventTemplate} from "./view/trip-event";
 import {createTripEditEventTemplate} from "./view/trip-edit-event";
-import {createTripCreateEventTemplate} from "./view/trip-create-event";
 import {createTripEventItemTemplate} from "./view/trip-event-item";
 
-const EVENT_COUNT = 3;
+import {generateTripEvent} from "./mock/trip-event";
+import {CITIES} from "./helpers/constants";
+
+const EVENT_COUNT = 15;
+
+const state = {
+  tabs: [
+    {
+      name: `Table`,
+      url: `#`,
+      isActive: true
+    },
+    {
+      name: `Stats`,
+      url: `#`,
+      isActive: false
+    }
+  ],
+
+  activeSort: `day`,
+
+  activeFilter: ``,
+
+  eventPoints: [],
+
+  tripInfo: {}
+};
+
+const generatedEventPoints = new Array(EVENT_COUNT).fill().map(generateTripEvent);
```

Comment:
Возможно тебе понравится другой способ генерации (без промежуточного fill):
```suggestion
const generatedEventPoints = Array.from({ length: EVENT_COUNT }, generateTripEvent);
```